react-els-app/src/components/templates/ProtectedTags.js

```diff
@@ -0,0 +1,16 @@
+import { useSelector } from 'react-redux';
+
+const ProtectedTags = ({ children, allowedRoles }) => {
+    const { isLoggedIn } = useSelector(state => state.persist.userAuthentication);
+
+    if(!isLoggedIn && (allowedRoles === 'guest')) {
```

Comment:
Create a json file that maps the allowedRoles
```js
roles: {
  GUEST: 'guest',
  AUTHTENTICATED: 'authenticated',
}
```

```suggestion
//   import roles from 'path/to/roles.json'

    if(!isLoggedIn && (allowedRoles === GUEST)) {
```